docker/shdb.cfg

```diff
@@ -0,0 +1,6 @@
+[SHDB_Settings]
```

Comment:
If this is considered like a INI file, you shouldn't leave whitespaces between key and value.

Reply:
Yes, thanks @sduenas . I will change that and merge!